pkg/utils/helper.go

```diff
@@ -11,6 +12,64 @@ import (
 	"strings"
 )
 
+// PublicAzureStorageEndpointSuffix defines default Storage Endpoint Suffix
+const PublicAzureStorageEndpointSuffix = "core.windows.net"
+
+// Azure defines Azure configuration
+type Azure struct {
+	Cloud string `json:"cloud"`
+}
+
+// AzureStackCloud defines Azure Stack Cloud configuration
+type AzureStackCloud struct {
+	StorageEndpointSuffix string `json:"storageEndpointSuffix"`
+}
+
+// IsAzureStackCloud returns true if the application is running on Azure Stack Cloud
+func IsAzureStackCloud() bool {
+	azureFile, err := RunCommandOnHost("cat", "/etc/kubernetes/azure.json")
+	if err != nil {
+		return false
+	}
+	var azure Azure
+	err = json.Unmarshal([]byte(azureFile), &azure)
+	if err != nil {
+		return false
+	}
+	cloud := azure.Cloud
+	return strings.ToUpper(cloud) == "AZURESTACKCLOUD"
+}
+
+// CopyCert saves the cert content to a new file inside the container
+func CopyCert() error {
+	cert, err := RunCommandOnHost("cat", "/etc/ssl/certs/azsCertificate.pem")
+	if err != nil {
+		return fmt.Errorf("Failed to retrieve cert content: %+v", err)
+	}
+	err = WriteToFile("/etc/ssl/certs/azsCertificate.pem", cert)
+	if err != nil {

```

Comment:
you could have a more concise code with 
`if err = WriteToFile("/etc/ssl/certs/azsCertificate.pem", cert); err != nil {`